mozilla-services/fx-crash-sig
# Switch to pyproject.toml (#97)

- Switch to pyproject.toml (#97)
- Update ruff configuration

To test:

1. create and activate a new virtual environment
2. run: `pip install -e '.[dev]'`
3. run: `fx-crash-sig --help` and make sure it prints help text
4. run: `make test` and make sure tests and linting pass
5. run: `python -m build` and make sure it generates a `.tar.gz` file and a `.whl` file
6. run: `check-manifest` and make sure it reports no errors

Fixes #97
